pom.xml

```diff
@@ -36,7 +36,7 @@
         <findbugs.version>1.3.9</findbugs.version>
         <guava.version>29.0-jre</guava.version>
         <immutables.version>2.9.0</immutables.version>
-        <jackson.version>2.13.3</jackson.version>
+        <jackson.version>2.13.2.2</jackson.version>
```

Comment:
Is it possible to add a comment here so we don't update it again when snyk opens a PR?

Reply:
Good suggestion, comment added.